Warn/error when `serve` is not pointing at an absolute path

Relative path fails:

```
$ sf-fx-runtime-nodejs serve ../buildpacks/buildpacks-nodejs/test/fixtures/simple-function/
{"level":50,"time":1628093075303,"pid":32502,"hostname":"rschneeman-ltm6.internal.salesforce.com","msg":"Could not load function: Could not load 'package.json' from project directory: Cannot find module '../buildpacks/buildpacks-nodejs/test/fixtures/simple-function/package.json'\nRequire stack:\n- /Users/rschneeman/Documents/projects/work/sf-fx-runtime-nodejs/dist/src/user-function.js\n- /Users/rschneeman/Documents/projects/work/sf-fx-runtime-nodejs/dist/src/index.js\n- /Users/rschneeman/Documents/projects/work/sf-fx-runtime-nodejs/bin/invoke.js"}
```

Absolute path works:

```
$ sf-fx-runtime-nodejs serve /Users/rschneeman/Documents/projects/work/buildpacks/buildpacks-nodejs/test/fixtures/simple-function/
```

--- a/src/index.ts
+++ b/src/index.ts
@@ -3,6 +3,7 @@ import { hideBin } from "yargs/helpers";
 import { loadUserFunctionFromDirectory } from "./user-function";
 import startServer from "./server";
 import logger from "./logger";
+import * as path from "path";
 
 yargs(hideBin(process.argv))
   .command(
@@ -30,7 +31,8 @@ yargs(hideBin(process.argv))
     (args) => {
       let userFunction;
       try {
-        userFunction = loadUserFunctionFromDirectory(args.projectPath);
+        const absolutePath = path.resolve(args.projectPath);
+        userFunction = loadUserFunctionFromDirectory(absolutePath);
       } catch (error) {
         logger.error("Could not load function: " + error.message);
         process.exit(1);

